neo/Consensus/ConsensusService.cs

```diff
@@ -136,13 +148,31 @@ private void Log(string message, LogLevel level = LogLevel.Info)
 
         private void OnChangeViewReceived(ConsensusPayload payload, ChangeView message)
         {
+            if (context.State.HasFlag(ConsensusState.CommitSent))
+                return;
```

Comment:
We do not have recover message. What's would happen if half of the CN was restarted when others are in this step

Reply:
Yeah, it has to have the regeneration message. I noticed the same thing as well, but believe they want to make the change in a separate PR on top of this.

Reply:
If it will come in other PR, it's fine

Reply:
I will save the state of the commit phase on the hard disk in the next pr. If the node restarts and finds the state of the commit phase last time, it will automatically recover from this.

Reply:
The was the only point not resolved here. However, it will be further discussed in the next PR to be merged into #547.